### PR TITLE
TAP 1.4 RA update

### PR DESCRIPTION
**Pull request by sendjainabhi**
Signed-off-by: Abhishek Jain <abhishekja@vmware.com>

TAP RA has been updated for TAP 1.4 version.

### CODE REVIEW

File: src/reference-designs/tap-architecture-planning.md [hunk 1]
```diff
@@ -314,7 +485,7 @@ Logging for Tanzu Application Platform is handled by the upstream Kubernetes int
 
 There are multiple ways to set up authentication in a Tanzu Application Platform deployment. You can manage authentication at the infrastructure level with your Kubernetes provider. VMware recommends Pinniped for integrating your identity provider into Tanzu Application Platform.
 
-To use Pinniped, see [Installing Pinniped on Tanzu Application Platform](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.3/tap/GUID-authn-authz-pinniped-install-guide.html) and [Login using Pinniped](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.3/tap/GUID-authn-authz-pinniped-login.html).
+To use Pinniped, see [Installing Pinniped on Tanzu Application Platform](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.4/tap/authn-authz-pinniped-install-guide.html) and [Login using Pinniped](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.4/tap/authn-authz-pinniped-login.html).
```
Comment (reshmarajkumar-vmw) [inline src/reference-designs/tap-architecture-planning.md:488]:
```suggestion
To use Pinniped, refer to [Installing Pinniped on Tanzu Application Platform](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.4/tap/authn-authz-pinniped-install-guide.html) and [Login using Pinniped](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.4/tap/authn-authz-pinniped-login.html).
```

File: src/reference-designs/tap-architecture-planning.md [hunk 2]
```diff
@@ -334,15 +505,12 @@ Two roles are for service accounts associated with the Tanzu Supply Chain:
 * `workload`
 * `deliverable`
 
-Refer [Tanzu Application Platform authorization](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.3/tap/GUID-authn-authz-overview.html) for more information.
-
-## CI/CD Pipelines
-Tanzu Application Platform supports Tekton pipelines using `tekton-pipelines package`. It allows developers to build, test, and deploy across cloud providers and on-premises systems. Refer [Tekton documentation](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.3/tap/GUID-tekton-tekton-about.html) for more information.
+Refer [Tanzu Application Platform authorization](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.4/tap/authn-authz-overview.html) for more information.
```
Comment (reshmarajkumar-vmw) [inline src/reference-designs/tap-architecture-planning.md:508]:
```suggestion
Refer to [Tanzu Application Platform authorization](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.4/tap/authn-authz-overview.html) for more information.
```

File: src/reference-designs/tap-architecture-planning.md [hunk 3]
```diff
@@ -334,15 +505,12 @@ Two roles are for service accounts associated with the Tanzu Supply Chain:
...
 
 ## Developer tools (Inner-Loop)
 
 Tanzu Application Platform allows developers to quickly build and test applications and provide many in-built developer friendly platform capabilities. To learn more about these capabilities, refer to [Tanzu Application Platform Developer Components](tap-architecture-dev-components.md).
 
 ## Deployment Instructions
 
-For instructions on how to deploy this reference design, see [Deploy multi-cluster Tanzu Application Platform profiles](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.3/tap/GUID-multicluster-installing-multicluster.html).
+For instructions on how to deploy this reference design, see [Deploy multi-cluster Tanzu Application Platform profiles](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.4/tap/multicluster-installing-multicluster.html).
```
Comment (reshmarajkumar-vmw) [inline src/reference-designs/tap-architecture-planning.md:516]:
```suggestion
For instructions on how to deploy this reference design, refer to [Deploy multi-cluster Tanzu Application Platform profiles](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.4/tap/multicluster-installing-multicluster.html).
```

**Review by reshmarajkumar-vmw — CHANGES_REQUESTED** [review]
Have made very few minor edits. The reason 'see' has been replaced with 'refer to' is to keep it consistent to the rest of the document. However, 'see' is the preferred word. This has to be changed globally across later on.

File: src/reference-designs/tap-architecture-planning.md (hunk 3 above)
Comment (sendjainabhi) [reply src/reference-designs/tap-architecture-planning.md]: I have accepted and changed in all places.

**Review by reshmarajkumar-vmw — APPROVED** [review]
Approved!

**Pull request merged**